build.gradle

```diff
@@ -37,46 +37,46 @@ ext {
     // Sdk and tools
     // Support library and architecture components support minSdk 14 and above.
     minSdkVersion = 21
-    targetSdkVersion = 29
-    compileSdkVersion = 29
+    targetSdkVersion = 30
```

Comment:
IMO, important ext needs to be in build.gradle(:app) for more and easy readability.

Reply:
Take a look please in google arch samples and code labs

Reply:
yes, I know they are doing the same you done so I wrote my opinion.
I just don't like switching between 2 files to see the versions specially `minSdk` and `targetSdk` :sweat_smile: